.rubocop.yml

```diff
@@ -18,12 +18,12 @@ Style/ClassAndModuleChildren:
   Enabled: false
 Style/DoubleNegation:
   Enabled: false
-Style/FileName:
+Naming/FileName:
```

Comment:
🗒️ I had to upgrade the rubocop gem to support `TargetRubyVersion: 2.5`. The new version migrated the names of some cops, hence the changes in these configs.